org.learningequality.Kolibri.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>org.learningequality.Kolibri</id>
+  <name>Kolibri</name>
+  <developer_name>Learning Equality</developer_name>
+  <summary>Offline education technology platform</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <url type="homepage">https://learningequality.org/kolibri/</url>
+  <url type="donation">https://learningequality.org/donate/</url>
+  <url type="help">https://kolibri.readthedocs.io/en/latest/</url>
+  <url type="bugtracker">https://github.com/learningequality/kolibri/issues</url>
+  <url type="translate">https://learningequality.org/translate/</url>
+  <description>
+    <p>Kolibri makes high quality education technology available in low-resource communities such as rural schools, refugee camps, orphanages, non-formal school systems, and prison systems. Running on a wide variety of hardware, it enables offline access to a rich set of content and tools to support learners, coaches, and administrators.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Browsing content channels in Kolibri</caption>
+      <image type="source" width="1544" height="806">https://learningequality.org/static/img/kickstarter/Kolibri-channels.png</image>
+    </screenshot>
+  </screenshots>
+  <categories>
+    <category>Education</category>
+    <category>Learning</category>
+    <category>Multimedia</category>
+    <category>Videos</category>
+    <category>Quiz</category>
+  </categories>
+  <icons>
+    <icon type="remote" height="128" width="128">https://raw.githubusercontent.com/learningequality/kolibri-installer-flatpak/master/src/icon-128.png</icon>
+    <icon type="local" height="128" width="128">/app/share/icons/hicolor/128x128/apps/org.learningequality.Kolibri.png</icon>
+    <icon type="remote" height="64" width="64">https://raw.githubusercontent.com/learningequality/kolibri-installer-flatpak/master/src/icon-64.png</icon>
+    <icon type="local" height="128" width="128">/app/share/icons/hicolor/64x64/apps/org.learningequality.Kolibri.png</icon>
+  </icons>
```

Comment:
Same as the categories.

Reply:
Ok, perfect. These are already installed and then referenced in the desktop file, so I'll just remove them from here.